easybuild/easyconfigs/t/timm/timm-1.0.8-foss-2023a-CUDA-12.1.1.eb

```diff
@@ -0,0 +1,45 @@
+easyblock = 'PythonBundle'
+
+name = 'timm'
+version = '1.0.8'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://huggingface.co/docs/timm'
+description = """
+timm is a library containing SOTA computer vision models, layers, utilities,
+optimizers, schedulers, data-loaders, augmentations, and training/evaluation
+scripts.  It comes packaged with >700 pretrained models, and is designed to be
+flexible and easy to use.
+"""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+dependencies = [
+    ('Python', '3.11.3'),
+    ('CUDA', '12.1.1', '', SYSTEM),
+    ('PyTorch', '2.1.2', versionsuffix),
+    ('PyYAML', '6.0'),
+    ('tqdm', '4.66.1'),
+    ('torchvision', '0.16.0', versionsuffix),
+    ('Safetensors', '1.4.3'),
+]
+
+builddependencies = [
+    ('maturin', '1.4.0', '-Rust-1.75.0'),  # for safetensors
```

Comment:
@laraPPr This build dependency can also be removed since `Safetensors` is a dependency now